source/features/git-checkout-pr.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import ClippyIcon from 'octicon/clippy.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getCurrentBranch} from '../github-helpers';
+
+function handleMenuOpening({delegateTarget: dropdown}: delegate.Event): void {
+	dropdown.classList.add('rgh-git-checkout'); // Mark this as processed
+	const [, user, repository] = select<HTMLAnchorElement>('.commit-ref.head-ref a')!.pathname.split('/', 3);
+	const isLocalPr = select('.user-select-contain.head-ref a')!.childElementCount === 1;
+	const tabContainer = select('tab-container', dropdown)!;
+	tabContainer.style.width = 'fit-content';
```

Comment:
I mean our new tabpanel should have this, not the container. I don't think the container needs to be touched at all

Reply:
If I don't add it to the container.

![image](https://user-images.githubusercontent.com/16872793/96589102-7026c600-12b2-11eb-9dd2-a36962824d25.png)

Reply:
Should I do?
```suggestion
	const currentWidth = looseParseInt(tabContainer.style.width);
	tabContainer.style.width = String(currentWidth + 90) + 'px';
```

Reply:
Your suggestion could be simplified:

```suggestion
	tabContainer.style.width = `${tabContainer.clientWidth + 90}px`;
```

Reply:
```suggestion
```

It looks like this for me, there's a scroll container there:

<img width="463" alt="" src="https://user-images.githubusercontent.com/1402241/96624991-2ef2de00-12d3-11eb-94fb-222ec1cbf326.png">


I don't want to make it even longer than my current 442px (I have CodeSpaces enabled)

Reply:
> If I don't add it to the container.
> 
> ![image](https://user-images.githubusercontent.com/16872793/96589102-7026c600-12b2-11eb-9dd2-a36962824d25.png)

How did you even get this screenshot? .UnderlineNav has a horizontal scroll by default:

<img width="149" alt="" src="https://user-images.githubusercontent.com/1402241/96625200-79745a80-12d3-11eb-8a18-865e11662688.png">

Reply:
![image](https://user-images.githubusercontent.com/16872793/96650384-cde1ff00-1300-11eb-94af-d318f1e22800.png)

This is how it look by me now

Reply:
You can add a `min-width: 350px` or something that looks good under 400